src/components/Properties.astro

```diff
@@ -1,8 +1,49 @@
-<div class="my-6">
+---
+const { heading = 'Attributes' } = Astro.props;
+---
+
+<div data-attributes class="my-6">
+  <div class="flex justify-between items-center not-prose py-2">
+    <h3 class="scroll-mt-10 type-subtitle-1">{ heading }</h3>
+    <button data-expand-button class="text-content-tertiary type-overline">
+      Expand all
+    </button>
+  </div>
   <ul
+    data-attribute-list
     role="list"
-    class="m-0 list-none divide-y divide-zinc-900/5 p-0"
+    class="m-0 list-none border-y border-zinc-900/5 divide-y divide-zinc-900/5 p-0 pt-4"
```

Comment:
Suggestion: Can the border colour be our colour token `border-outline-transparent`?

Reply:
Thats even more transparent than we currently have

Reply:
That's what's in the design 🤷‍♀️